steps.go

```diff
@@ -58,6 +58,7 @@ func initSteps(ctx StepAdder) {
 	ctx.Step("^I put the following documents in the corresponding collections:$", IPutDocumentsInMongoColl)
 	ctx.Step(`^the following document IDs should exist in the corresponding mongo collections:$`, TheFollowingDocumentsShouldExistInMongoCollections)
 	ctx.Step(`^the following documents should match the following files:$`, TheFollowingDocumentsShouldMatchTheFollowingFiles)
+	ctx.Step(`^I ignore from all mongo documents:$`, IIgnoreFromAllMongoDocuments)
```

Comment:
This step isn't needed since we're going down the inclusion route.

Reply:
Done :)